### ISSUE TITLE
Project is not compiling after following contributing guidelines

### ISSUE THREAD

**Issue by ferrannp** [pedronauck/docz#511]
## Bug Report

**Describe the bug**

I wanted to contribute but following [CONTRIBUTING.md](../CONTRIBUTING.md) does not work. The project is not compiling. I tried it on `master` and `dev`.

**To Reproduce**

1. Clone the repo
2. `yarn`
3. `yarn bs`
4. `yarn packages`

```shell
⠹ Compiling src/index.tsx...
Error: src/index.tsx:2:48 - error TS2307: Cannot find module 'docz'.
2 import { theme, DocPreview, ThemeConfig } from 'docz'
                                                 ~~~~~~
src/index.tsx:23:8 - error TS7006: Parameter 'config' implicitly has an 'any' type.

23       {config => (
          ~~~~~~

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run build stderr:
error Command failed with exit code 1.

lerna ERR! yarn run build exited 1 in 'docz-theme-default'
```
**Environment**

* OS: OSX 10.14.1
* Node/npm version: 10.11.0/6.4.1

**Comment by pedronauck** [comment]
I think that this is happening because of the cycled dependency of `docz` and `docz-theme-default` 😕 What you can do is `cd ./packages/docz && yarn build` then you can run `yarn packages:build`

**Comment by pedronauck** [comment]
Try this, then tell me

**Comment by pedronauck** [comment]
Fixed on [v0.12.17](https://github.com/pedronauck/docz/releases/tag/v0.12.17) ✅
